API/schema/message-schema.txt

```diff
@@ -1,31 +1,54 @@
+//Database
 //Our message system will limit to 2 interlocutors
 {
     //The ultimate storage of all the conversations between users
     chatStorage: [
         //One conversation between 2 people, including all their chat history and messages
         {
             //This array includes the names of the two interlocutors
-            interlocutors: ["name1","name2"], 
+            interlocutors: [str, str], (alphabetical)
 
             //This array stores all the messages between the two interlocutors
             chatHistory: [
-                {"chatTime": "18:30 2020/08/19", "chatSender": "name1", "chatContent": "HI Bill!"},
-                {"chatTime": "18:34 2020/08/19", "chatSender": "name2", "chatContent": "HI, Kerwin!"},
-                {"chatTime": "18:35 2020/08/19", "chatSender": "name2", "chatContent": "What's up?"}
+                {"chatTime": datetime, "chatSender": str (interlocuters[i]), "chatContent": str},
```

Comment:
Although I write `datetime` here, it will only be a `str` in `json` format. Will need to add datetime parsing when reading in data (override `JSON.parse()`)